src/Common/src/System/Diagnostics/Debug.Unix.cs

```diff
@@ -8,108 +8,118 @@ namespace System.Diagnostics
 {
     // Intentionally excluding visibility so it defaults to internal except for
     // the one public version in System.Diagnostics.Debug which defines
-    // another version of this partial class with the public visibility 
+    // another version of this partial class with the public visibility
     static partial class Debug
     {
-        private static string NewLine => "\n"; 
+#if !MONO
```

Comment:
So you mean in `src/Common/src/System/Diagnostics/Debug.cs` have `private static string NewLine => PlatformHelper.IsWindows ? "\n\r" : "\n"`? The problems with this approach would be that, first PlatformHelper is a Mono specific class (it's in `PlatformHelper.Mono.cs`), and second you cannot do the same thing for `s_logger` as `UnixLogger` and `WindowsLogger` are not compiled on all platforms, and making them compile on all platforms mean that you would need to have the Unix and Windows Interop on all platforms, etc.